Items/uraniumHorseshoe.lua

```diff
@@ -0,0 +1,33 @@
+local sprite = Resources.sprite_load(NAMESPACE, "UraniumHorseshoe", path.combine(PATH, "Sprites/Items/uraniumHorseshoe.png"), 1, 14, 17)
+local sprite_footstep = Resources.sprite_load(NAMESPACE, "UraniumHorseshoeFootstep", path.combine(PATH, "Sprites/Items/Effects/horseshoeFootstep.png"), 7, 3, 0)
+
+local parHorseshoe = Particle.new(NAMESPACE, "parHorshoe")
+parHorseshoe:set_sprite(sprite_footstep, true, true, false)
+parHorseshoe:set_life(56, 56)
+
+local horseshoe = Item.new(NAMESPACE, "uraniumHorseshoe")
+horseshoe:set_sprite(sprite)
+horseshoe:set_tier(Item.TIER.common)
+horseshoe:set_loot_tags(Item.LOOT_TAG.category_utility)
+
+horseshoe:clear_callbacks()
+horseshoe:onStatRecalc(function(actor, stack)
+	actor.pHmax = actor.pHmax + 0.28 * stack
```

Comment:
A quick comment explaining these numbers so future people reading the code understand how we got them